src/packages.builds

```diff
@@ -54,10 +54,6 @@
     <Project Include="System.ServiceProcess.ServiceController\pkg\*.pkgproj">
       <AdditionalProperties>$(AdditionalProperties)</AdditionalProperties>
     </Project>
-    <!-- Servicing support package to include the changes required for 4.7.1 -->
-    <Project Include="$(MSBuildThisFileDirectory)..\pkg\NETStandard.Library.NETFramework\NETStandard.Library.NETFramework.builds">
```

Comment:
I deleted this since @joperezr already serviced it and since we don't ship it stable and don't publish it to NuGet, then there is no need of re-service it.

Reply:
Yup, removing it is fine.

Reply:
@joperezr can we please make sure that we capture the symbols package for the build we are releasing to ensure we can get them indexed (assuming they aren't already).